### ISSUE TITLE
Our fleek build command for reproducible builds is too wordy and brittle

### ISSUE THREAD

**Issue by davidmurdoch** [trufflesuite/trufflesuite.com#1210]
Should the fleek build command be `pip3 install -r requirements.txt && mkdocs build` instead of:
![image](https://user-images.githubusercontent.com/187813/160896378-561d1daf-ccc6-4f89-934d-87a27f521ab4.png)

**Comment by gnidan** [comment]
Yes

**Comment by gnidan** [comment]
This might be done already - will need to check on the Fleek UI side.

**Comment by gnidan** [comment]
@kevinbluer confirms that this is done!
